Gordon360/Static Classes/Names.cs

```diff
@@ -29,6 +29,7 @@ public static class Resource
         public const string CHECKIN = "Info relating to a student's Academic Check-In";
         public const string SHIFT = "A shift that a student has worked";
         public const string CLIFTON_STRENGTHS = "A student's uploaded clifton strengthsfinder results";
+        public const string RECIM = "General/Admin RecIM resource";
```

Comment:
```suggestion
        public const string RECIM = "A general or admin RecIM resource";
```
Consistent naming with the rest of the resources.